src/application/services/useNote.ts

```diff
@@ -0,0 +1,48 @@
+import { Ref, ref } from 'vue';
+import Note from '../../domain/entities/Note';
+import { noteService } from '../../domain';
+
+/**
+ * Note hook state
+ */
+interface UseNoteComposableState {
+  /**
+   * Note ref
+   */
+  note: Ref<Note | null>;
+
+  /**
+   * Load note
+   *
+   * @param id
+   */
+  load: (id: number) => Promise<void>;
+}
+
+/**
+ * useNote hook
+ *
+ * @returns { UseNoteComposableState } - Note hook state
+ */
+function useNote(): UseNoteComposableState {
+  /**
+   * Note ref
+   */
+  const note = ref<Note | null>(null);
+
+  /**
+   * Get note
+   *
+   * @param id - Note id
+   */
+  const load = async (id: number): Promise<void> => {
+    note.value = await noteService.getNoteById(id);
+  };
+
+  return {
+    note,
+    load,
+  };
+}
+
+export default useNote;
```

Comment:
I'm not sure about export default, it depends of project, but if you use default, you can rename function

https://stackoverflow.com/questions/33611812/export-const-vs-export-default-in-es6